### ISSUE TITLE
pressing 'up' to edit a message doesn't jump the edit box into view if you're scrolled up.

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#16402]
i'd expect when you focus the edit box for it to become visible

**Comment by t3chguy** [comment]
Cannot reproduce, it scrolls into view for me.


https://user-images.githubusercontent.com/2403652/114539316-4a202400-9c4c-11eb-8299-4f9173fe0acf.mov

**Comment by SimonBrandner** [comment]
I can repro this if I am scrolled up enough that the message isn't in the paginated area (I hope that's how it's called)

**Comment by t3chguy** [comment]
So if the app doesn't have a recent message from you loaded then that's expected, but the issue there isn't about scrolling but that it just does nothing, no?

**Comment by SimonBrandner** [comment]
I am not sure I understand, steps to repro my issue:

+ Go to a room with a bunch of traffic
+ Send a message
+ Scroll up until the message isn't in the loaded area
+ Press up arrow - you won't get scrolled down to see the message

If this isn't what the issue is about, I misunderstood - sorry

**Comment by t3chguy** [comment]
Right, if you scroll enough for that event to be paged-out of memory then that is expected, we only check the local timeline and not ask the server to find the message to edit

**Comment by SimonBrandner** [comment]
Ah, that makes sense. This means I can't repro either

**Comment by t3chguy** [comment]
Closing due to unable to reproduce
